documentation-site/pages/theming/custom-themes.mdx

```diff
@@ -35,14 +35,14 @@ All themeable values are available within components that provide the `overrides
 
 To learn more on how overrides work, check out [Better Reusable React Components with the Overrides Pattern](https://medium.com/@dschnr/better-reusable-react-components-with-the-overrides-pattern-9eca2339f646) article.
 
-## The `ThemeProvider`
+## The `BaseProvider`
 
-With the `ThemeProvider`, you can apply a theme to your application globally or locally. By default, Base Web ships with a light and a dark theme.
+With the `BaseProvider`, you can apply a theme to your application globally or locally. By default, Base Web ships with a light and a dark theme.
```

Comment:
How about changing it in a way that says `use BaseProvider to apply a theme globally and ThemeProvider to apply it locally`?

Reply:
is there a difference between the two? if not, I think we should tell folks to use the same thing everywhere to keep things simple

Reply:
The difference is that BaseProvider adds more Providers than a single Theme one. We don't want another LayersProvider to be added in a separate part of the app, unless it's intended.

Reply:
oh, that makes sense - let me update this PR